bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseActuatorHandler.java

```diff
@@ -296,10 +296,10 @@ public void handleRemoval() {
 
     @Override
     public void dispose() {
-        ScheduledFuture<?> localRefreshJob = refreshJob;
```

Comment:
Any reason why this is reverted ? This is setup as many other bindings. I would opt to leave it as it was.

Reply:
I didn't revert anything, at least not on purpose as I didn't check #14023 when creating this pull request.

I refactored this for consistency as I checked all references of "scheduler."

I personally prefer explicit `this.` references over having different naming like:
```java
String localHost = host;
```

This eliminates any kind of naming confusion like in my example. Additionally, it makes it clear that we are coping the reference from an instance variable, rather than for example another local variable. Thus, you can clearly see what the code does without having to follow the reference.

Reply:
I personally find the same name variable from diffent scopes only more confusing. Especially when the `this` preffix is optional (not forced by coding style) in most context and not consistently used everywhere. 

Either way is good from functional perspective, within the openhab bindings i can see both conventions are used together.

That being said, this code change is only changing this convention and not fixing any bug. So i can only agree to move on :-)

Reply:
> I personally find the same name variable from diffent scopes only more confusing. Especially when the `this` preffix is optional (not forced by coding style) in most context and not consistently used everywhere.

In this case it's not optional because of same name but different scope. 🙂 I agree that enforcing `this.` everywhere would improve readability and consistency. This is a broader topic and I have kind of settled with the current style and naming conventions as every project has their own, and there are many opinions etc., so I won't start this discussion.